bin/rspecq

```diff
@@ -100,19 +106,21 @@ if opts[:report]
     build_id: opts[:build],
     timeout: opts[:report_timeout],
     redis_host: opts[:redis_host],
+    fail_fast: opts[:fail_fast]
   )
 
   reporter.report
 else
   worker = RSpecQ::Worker.new(
     build_id: opts[:build],
     worker_id: opts[:worker],
-    redis_host: opts[:redis_host]
+    redis_host: opts[:redis_host],
   )
 
   worker.files_or_dirs_to_run = ARGV[0] if ARGV[0]
   worker.populate_timings = opts[:timings]
   worker.file_split_threshold = opts[:file_split_threshold]
   worker.max_requeues = opts[:max_requeues]
+  worker.fail_fast = opts[:fail_fast]
```

Comment:
Let's not set this if it's not necessary, like we do with `files_or_dirs_to_run` above:

```ruby
worker.fail_fast = opts[:fail_fast] if opts[:fail_fast]
```

I know it's slightly more verbose and the result is the same, but I believe it's more future-proof because it makes it harder to screw up something by passing an unexpected value here (e.g. `true`, `false` etc.)